model/mobile_trigger.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type MobileTrigger struct {
+	PluginId string `json:"id"`
+	Location string `json:"location"`
+	Trigger  string `json:"trigger"`
+	//	Extra    interface{} `json:"extra"`
+	Extra string `json:"extra"` // Should use interface{} to be able to create different extras for different locations
+}
```

Comment:
I like the `Scope` idea from Maria, but that could also be done in a different PR. 1/5 that we should rather add it now, as it's easy to implment.